v22.2/restore.md

```diff
@@ -467,6 +469,24 @@ RESTORE TABLE movr.users FROM LATEST IN 's3://{bucket_name}?AWS_ACCESS_KEY_ID={k
 
 For more detail on using this option with `BACKUP`, see [Incremental backups with explicitly specified destinations](take-full-and-incremental-backups.html#incremental-backups-with-explicitly-specified-destinations).
 
+#### Validate a backup with restore
```

Comment:
hrm, the text blocks below look like a lot of copying and pasting-- do we really need them?

Reply:
I would say no. Just trying to cover the bases. But I think having the options in the respective SQL pages option table and linking to the Backup Valid. page is enough. I will drop these.